src/Components/Settings/Settings.js

```diff
@@ -73,6 +73,7 @@ const Settings = () => {
           <option value={'pianoDreams'}>Piano Dreams</option>
           <option value={'levelUp'}>Level Up</option>
           <option value={'birdChord'}>Bird Chord</option>
+          <option value={'random'}>Randomize My Sounds</option>
```

Comment:
Ooooh very nice! I was imagining a button or checkbox or something but I like this much better.

Reply:
OOh i like this too! Having it in the dropdown is cool